examples/correlated_ou/Experiment.toml

```diff
@@ -2,30 +2,20 @@
 project           = "CliMAgen.jl"
```

Comment:
Do examples/correlated_ou/ files need to be in this PR?

Reply:
oh jk, I understand!

Reply:
I think we can remove the "_old" files. We never really got the old version going (generating videos with regressions/conditional on previous steps)

Reply:
also any old data loader for this data.